DataTool/ToolLogic/List/Misc/ListWorkshop.cs

```diff
@@ -212,6 +226,17 @@ public class WorkshopDefinition {
             public IEnumerable<WorkshopParameter> Parameters;
         }
 
+        public class ExtensionVal {
+            public teResourceGUID DisplayName_Id;
+            public string DisplayName;
+            public teResourceGUID STUId;
```

Comment:
What is STUId? I assume it's a 01C file, if so can we call it Virtual01C

Reply:
yep